src/modules/content-s3/s3.c

```diff
@@ -81,6 +81,11 @@ int s3_init (struct s3_config *cfg, const char **errstr)
 {
     S3Status status = S3_initialize ("s3", S3_INIT_ALL, cfg->hostname);
 
+    if (cfg->is_virtual_host > 0)
```

Comment:
Drop > 0 here and the next one.  The test should be "is nonzero" and that's the same as "true" in C.